app/controllers/users_controller.rb

```diff
@@ -15,4 +15,8 @@ def update
       render :edit
     end
   end
+
+  def bids_placed
```

Comment:
I just am implementing what was in the issue. I think my problem with just using `/bids` is that this is not a list of all bids, but just those scoped to the user. I personally would prefer `/account/bids` to `bids_placed` though